pkgs/applications/graphics/imv/default.nix

```diff
@@ -1,44 +1,40 @@
-{ stdenv, fetchFromGitHub
-, freeimage, fontconfig, pkgconfig
-, asciidoc, docbook_xsl, libxslt, cmocka
-, librsvg, pango, libxkbcommon, wayland
-, libGLU, icu
-}:
+{ stdenv, fetchFromGitHub, freeimage, fontconfig, pkgconfig, asciidoc
+, docbook_xsl, libxslt, cmocka, pango, libxkbcommon, wayland, libGLU, icu
+, libtiff, libpng, libjpeg_turbo, librsvg, libheif, netsurf }:
 
 stdenv.mkDerivation rec {
   pname = "imv";
   version = "4.1.0";
 
   src = fetchFromGitHub {
-    owner  = "eXeC64";
-    repo   = "imv";
-    rev    = "v${version}";
+    owner = "eXeC64";
+    repo = "imv";
+    rev = "v${version}";
     sha256 = "0gk8g178i961nn3bls75a8qpv6wvfvav6hd9lxca1skaikd33zdx";
   };
 
-  preBuild = ''
-    # Version is 4.0.1, but Makefile was not updated
-    sed -i 's/echo v4\.0\.0/echo v4.0.1/' Makefile
-  '';
-
-  nativeBuildInputs = [
-    asciidoc
-    cmocka
-    docbook_xsl
-    libxslt
-  ];
+  nativeBuildInputs = [ asciidoc cmocka docbook_xsl libxslt ];
 
   buildInputs = [
     freeimage
+    icu
     libGLU
+    libjpeg_turbo
     librsvg
     libxkbcommon
+    netsurf.libnsgif
     pango
     pkgconfig
     wayland
-    icu
   ];
 
+  preBuild = ''
```

Comment:
This file appears to `included` in the Makefile, so you should be able to simply use `makeFlags` to change these.

Reply:
Thanks!  I tried `installFlags` which did not work, with `makeFlags` it does :) Fixed.